.gitignore

```diff
@@ -10,5 +10,6 @@ build/**
 documentation/slimerjs-*/**
 documentation/api/**
 documentation/download/*.zip
+documentation/js/cytoscape.min.js
```

Comment:
```suggestion

```

Reply:
It's nice to have this checked in so that you can access the docs right away by checking out the repo (e.g. a tag for an old version).